client/src/components/EventForm/index.jsx

```diff
@@ -82,6 +83,43 @@ class EventForm extends Component {
     return errorMessages;
   }
 
+  createEventDataObj() {
+    let eventData = {
+      name: this.state.title,
+      lat: this.state.lat,
+      long: this.state.long,
+      description: this.state.description,
+      category: this.state.category,
+      start_date: moment(
+        [this.state.start_date, this.state.start_time].join(' '),
+        'D MMMM, YYYY HH:mm'
+      )._d.toJSON(),
+    };
+
+    if (this.state.end_date && this.state.end_time) {
+      eventData.end_date = moment(
+        [this.state.end_date, this.state.end_time].join(' '),
+        'D MMMM, YYYY HH:mm'
+      )._d.toJSON();
+
+      if (eventData.end_date < eventData.start_date) {
+        this.setState({
+          error: {
+            ...this.state.error,
+            end_date: 'End date can\'t be later than start',
+          }
+        });
+        return 1;
+      };
+    };
+
+    const tag_arr = window.$('#chips-tags').material_chip('data');
```

Comment:
I think tag_arr not readable good

Reply:
`arr` is a common abbreviation for an array, like temp for temporary or dict for a dictionary. Or do you mean something else?

Reply:
Maybe it's fine but for future need ask Serhiy